src/control/controlproxy.h

```diff
@@ -48,28 +48,21 @@ class ControlProxy : public QObject {
         // requested: Queued -> COP = Queued / SCO = Auto
         // requested: BlockingQueued -> Assert(false)
 
-        auto copSlot = &ControlProxy::slotValueChangedAuto;
-        Qt::ConnectionType copConnection;
         Qt::ConnectionType scoConnection;
         switch(requestedConnectionType) {
         case Qt::AutoConnection:
-            copConnection = Qt::AutoConnection;
+            [[fallthrough]];
+        case Qt::QueuedConnection:
             scoConnection = Qt::AutoConnection;
             break;
         case Qt::DirectConnection:
-            copSlot = &ControlProxy::slotValueChangedDirect;
-            copConnection = Qt::DirectConnection;
             scoConnection = Qt::DirectConnection;
             break;
-        case Qt::QueuedConnection:
-            copSlot = &ControlProxy::slotValueChangedQueued;
-            copConnection = Qt::QueuedConnection;
-            scoConnection = Qt::AutoConnection;
-            break;
```

Comment:
The code triggered this warning: https://github.com/KDE/clazy/blob/master/docs/checks/README-lambda-unique-connection.md

(see the commit message of https://github.com/mixxxdj/mixxx/pull/2446/commits/1aae2296f67af658c61f37cb5d8c4090878a69f4).

Reply:
Ah, so this is a false positive from Clazy :-/ 
Please add a comment that Clazy requires to give directly a member fuction in case of Qt::UniqueConnection.

Reply:
Done.